src/platform/datapath_raw.h

```diff
@@ -273,7 +273,7 @@ CxPlatSockPoolUninitialize(
 // conjunction with the hash table lookup, which already compares local UDP port
 // so it assumes that matches already.
 //
-inline
+static inline
```

Comment:
Why?

Reply:
as far as I remember there is compiler bug